.github/workflows/eval.yml

```
@@ -0,0 +1,88 @@
+name: Check eval on pull request
+
+on:
+  - workflow_dispatch
+  - push
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    name: eval-check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        system: [x86_64-linux, aarch64-linux, aarch64-darwin, x86_64-darwin]
+    steps:
+      - name: Set up Git
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "GitHub Actions"
```

One thing I've run into in the past is "global" can cause problems for other people.
Might it be worth switching to local (like this):
```suggestion
          git config --local user.name "GitHub Actions"
```
to not cause problems for other people?

---

It is "global" for the GH action run. So it doesn't really matter, I think